Affiliates.html

```diff
@@ -0,0 +1,65 @@
+﻿<!doctype html>
+<html>
+<head>
+    <title>Affiliates</title>
+    <meta charset="utf-8">
+    <script src="https://code.jquery.com/jquery-3.1.0.min.js"></script>
+    <!-- Normalize CSS -->
+    <link rel="stylesheet" href="CSS/Normalize.css" />
+    <!-- Latest compiled and minified CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.2/css/bootstrap.min.css">
+    <!-- Optional theme -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.2/css/bootstrap-theme.min.css">
+    <link rel="stylesheet" href="CSS/Site.css">
+    <link rel="stylesheet" href="CSS/public_documents.css">
+    <script>
+        $(function () {
+            $('#header').load("header.html");
+            $('#footer').load("footer.html");
+        });
+    </script>
+</head>
+<body>
+    <!-- This gets populated with the header.html using Javascript -->
+    <div id="header"></div>
+    <div class="container">
+        <div>
+            <h1>
+                Material from Affiliates
+            </h1>
+        </div>
+        <div>
+            <p>
+                A year into the formation of OMA, significant consolidation has been achieved with the integration of the WAP Forum, Location Interoperability Forum (LIF), SyncML Initiative, Multimedia Messaging Interoperability Forum (MMS-IOP), Wireless Village, Mobile Gaming Interoperability Forum (MGIF), and the Mobile Wireless Internet Forum (MWIF)
+
+                The work of these organizations continues in various Working Groups within OMA. Original documents created in each organization prior to the integration into OMA can be found by clicking on the name of the Affiliate:
+            </p>
+        </div>
+
+        <td>
+            <ul>
+                <li>
+                    <a target="_blank" href="http://openmobilealliance.github.io/Affiliates/LIF">Location Interoperability Forum (LIF)</a>

```

Comment:
Consider not making these 'target="_blank"'

Best practice is that opening things is new tabs should be limited for when navigating away from this particular site to a different one. If the navigation is within the same site, then just navigate there normally. 

The end-user can choose themselves to open a link in a new tab. By putting in target=blank, you're forcing them to do that and not giving them the option.